.github/workflows/build-firmware.yml

```
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - main
+    tag:
```

actionlint (1.7.8)

unexpected key "tag" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)